src/Form/Control/Multiline.php

```diff
@@ -420,13 +433,15 @@ public function setModel(Model $model, array $fieldNames = []): Model
      * Otherwise, form will try to save 'multiline' field value as an array when form is save.
      * $multiline = $form->addControl('multiline', [Multiline::class], ['never_persist' => true])
      */
-    public function setReferenceModel(Model $refModel, string $linkByFieldName, array $fieldNames = []): Model
+    public function setReferenceModel(string $refModelName, Model $modelEntity = null, array $fieldNames = []): Model
```

Comment:
search `setReferenceModel`, docs needs to be adjusted.

In general, do we need to pass `Model` at all?

Reply:
Doc updates. In general, the Model entity will be set using the model set in Form. But there might be cases where you need to pass it with the method.